### ISSUE TITLE
Game executable name does it have to be a profile name?

### ISSUE THREAD

**Issue by Krzysztof23** [adirh3/Sonar.AutoSwitch#14]
Is it possible to add the upload of own profile icons or at least add displaying the window name as a replacement if the name of the executable file is empty, alternatively, is it possible to add a field to fill in titled profile name?

I have several game profiles with their name empty and constantly checking what has been saved in them can be a bit of a pain

in addition, I would like to ask if it is possible to extend the capabilities of the application with the ability to detect several profiles at once, e.g. 1 for playing and 1 for listening to music at the same time?
![Zrzut ekranu 2023-08-26 201931](https://github.com/adirh3/Sonar.AutoSwitch/assets/67221783/ee8a3f53-1a2c-4ca6-871a-82ee73e777f7)

**Comment by Mativec** [comment]
Yes ! I just contributed so the app display the window name by default and the exec if missing
